src/SprykerEco/Zed/NewRelic/Communication/Console/RecordDeploymentConsole.php

```diff
@@ -53,13 +53,13 @@ protected function configure(): void
 
         $this->addArgument(
             self::ARGUMENT_USER,
-            InputArgument::OPTIONAL,
+            InputArgument::REQUIRED,
             self::ARGUMENT_USER_DESCRIPTION
         );
 
         $this->addArgument(
             self::ARGUMENT_REVISION,
-            InputArgument::OPTIONAL,
+            InputArgument::REQUIRED,
```

Comment:
Same as above.

Reply:
Revision is the only required parameter by new-relic API. Therefore if someone is using the record deployment, he still needs to send the first (required), the second (optional) and this (third one) to record a deployment. 
It's not a BC breaks, because it cannot work if we send only the first parameter anyway.

Reply:
ok